go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go

```diff
@@ -204,8 +199,7 @@ func NewTxThrottler(env tabletenv.Env, topoServer *topo.Server) TxThrottler {
 		config:           config,
 		topoServer:       topoServer,
 		throttlerRunning: env.Exporter().NewGauge(TxThrottlerName+"Running", "transaction throttler running state"),
-		topoWatchers:     env.Exporter().NewGaugesWithSingleLabel(TxThrottlerName+"TopoWatchers", "transaction throttler topology watchers", "cell"),
-		healthChecksReadTotal: env.Exporter().NewCountersWithMultiLabels(TxThrottlerName+"HealthchecksRead", "transaction throttler healthchecks read",
+		topoWatchers:     env.Exporter().NewGaugesWithSingleLabel(TxThrottlerName+"TopoWatchers", "DEPRECATED: transaction throttler topology watchers", "cell"), healthChecksReadTotal: env.Exporter().NewCountersWithMultiLabels(TxThrottlerName+"HealthchecksRead", "transaction throttler healthchecks read",
```

Comment:
Looks like a newline here was removed which makes the diff a bit confusing? 

Also, what is the general value / policy around metrics deprecations? Is it useful / expected for folks to have empty metrics? Does that have value over removing the metric?

Reply:
I'll add back the newline. I went back and forth between deleting the metric versus deprecating. We need to deprecate so that people's existing metrics pipelines don't break without warning.

Reply:
I've added a line in the release notes for these.

Reply:
> We need to deprecate so that people's existing metrics pipelines don't break without warning.

But doesn't a metric that flatlines break things anyway? I think it's also possible to argue that having a metric go to zero is more confusing / problematic than removing it as a signal to the user that the value is (no longer) useful? I think this is tricky anyway, we can argue both ways but I don't think this is as clear cut as say a command line flag that is deprecated?

Reply:
Honestly I don't know if anyone is even using this metric. It's not much effort to keep it deprecated and delete it in the next release, so I ended up choosing that as the "safer" approach.